azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/action/ChatAction.kt

```diff
@@ -3,4 +3,9 @@
 
 package com.azure.android.communication.ui.chat.redux.action
 
-internal sealed class ChatAction : Action
+internal sealed class ChatAction : Action {
+    class startChat : ChatAction()
```

Comment:
Lower case mixed with Upper case (s should be S)

Reply:
Thanks Adm. done